### PR TITLE
feat: add skeleton for browse depositions page

### PR DESCRIPTION
**Pull request by kne42**
#773

Screenshot 2024-06-21 at 1 23 55 PM

### CODE REVIEW

**Comment by kne42** [comment]
https://dev-kira-1.cryoet.dev.si.czi.technology/browse-data/depositions

note: for some reason switching between tabs takes longer in dev link vs locally

**Review by Janeece — CHANGES_REQUESTED** [review]
@kne42 wow, i can't believe it's already this far! 
Note: in my review i'm assuming data & images in the depositions table, including what are listed depositions, is all placeholder - will not comment on this

Feedback:
- [x] (1) Nice to Have - while we are working on this, can we fix the spacing for both tabs to be accurate to the design? The space between the tab label and the underline is too big. 
    - Note: if this is better to fix in the SDS refactor, then ignore
    - Screenshot 2024-06-24 at 10 29 04 AM

- [ ] (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size). 
    - If this is how other tables behave it should be fixed across all, but its a low priority. 
    - if this proves difficult to control/correct, we can prioritize other work first. 
    - Screenshot 2024-06-24 at 10 32 00 AM


- [ ] (3) Should have, not blocker - There is a tooltip when hovering over the date that repeats the date - please remove 
    - Screenshot 2024-06-24 at 10 20 27 AM

- [ ] (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead
    - Screenshot 2024-06-24 at 10 22 53 AM

**Comment by kne42** [comment]
> * [ ]  (1) Nice to Have - while we are working on this, can we fix the spacing for both tabs to be accurate to the design? The space between the tab label and the underline is too big.
fixed

> * [ ]  (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size).
>   
>   * If this is how other tables behave it should be fixed across all, but its a low priority.
>   * if this proves difficult to control/correct, we can prioritize other work first.

both of your bullets are true, so can't fix rn

> * [ ]  (3) Should have, not blocker - There is a tooltip when hovering over the date that repeats the date - please remove

fixed

> * [ ]  (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead

half-fixed – no longer wraps; however wont do ellipsis atm since the entire table scrolls when overflowing – do you have the link for the exact specs? i didnt see them in the linked figma

**Comment by Janeece** [comment]
> > * [ ]   (2) Should have, not blocker - space between keyphoto and deposition name shouldn't expand (keyphoto width should be fixed size).
> >   
> >   * If this is how other tables behave it should be fixed across all, but its a low priority.
> >   * if this proves difficult to control/correct, we can prioritize other work first.
> 
> both of your bullets are true, so can't fix rn

No problem, thanks for letting me know!

> > * [ ]   (4) Should have, not blocker - object shape column wraps overflow, should hide overflow instead
> 
> half-fixed – no longer wraps; however wont do ellipsis atm since the entire table scrolls when overflowing – do you have the link for the exact specs? i didnt see them in the linked figma

@kne42  The horizontal scroll works fine as a solution, it solves the main issue of avoiding text wrap on these lists.  
The spec ([here](https://www.figma.com/design/Rrr54QysCLa14vOnQcmejb/Janeece---Annotation-Depositions?m=dev&node-id=3227-62513)) has a min/max width recommendations but I know those aren't simple to implement exactly. It's not a blocker, so lmk if we won't be doing it & I'll approve the PR.

**Comment by kne42** [comment]
I've added the min/max widths but am keeping the scrolling for now since the object shape type column won't seem to size to its min no matter what I do

**Review by Janeece — APPROVED** [review]
@kne42 thanks for making the changes! looks great!

File: frontend/packages/data-portal/app/components/Tabs.tsx [hunk 1]
```diff
@@ -20,32 +20,34 @@ export function Tabs<T extends string>({
...
-              'transition-colors',
-            ),
-            selected: '!text-black',
-          }}
-          key={tab.value}
-          {...tab}
-        />
-      ))}
-    </MUITabs>
+    <div className="flex items-center">
```
[the remark this answers is not in the document]
Comment (kne42) [reply frontend/packages/data-portal/app/components/Tabs.tsx:23]: couldnt find another way to centre it; css is finnicky. the div keeps the parent flex from force-stretching our tabs which is the reason behind why the offset on the indicator was so large

**Pull request merged**